src/c2goto/library/pthread_lib.c

```diff
@@ -291,10 +295,25 @@ __ESBMC_HIDE:;
   return 0;
 }
 
+int pthread_mutex_initializer(pthread_mutex_t *mutex)
+{
+  // check whether this mutex has been initialized via
+  // PTHREAD_MUTEX_INITIALIZER
+#ifdef __APPLE__
+  if(mutex->__align == _PTHREAD_MUTEX_SIG_init)
```

Comment:
Shouldn't that be `mutex->sig`?

Reply:
We would need to change our implementation of the pthread_mutex_t to consider this comment.

See https://github.com/esbmc/esbmc/blob/master/src/c2goto/headers/pthreadtypes.hs#L112.